sys-process/rtirq/rtirq-20180209.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit systemd
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
+HOMEPAGE="http://www.rncbc.org/archive/#rtirq"
+
+SRC_URI="http://www.rncbc.org/archive/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	# Correct config file path.
```

Comment:
Please don't use `${PN}` to reference files in phase functions. This harms readability and makes it impossible to copy-paste stuff, e.g. to look at a particular file.

Reply:
Fixed in 46bf5ad584d30d440f1b05969eee5bfc8589c35f.